py-polars/tests/unit/interchange/test_roundtrip.py

```diff
@@ -123,7 +126,7 @@ def test_to_dataframe_pandas_zero_copy_parametric(df: pl.DataFrame) -> None:
 )
 def test_from_dataframe_pyarrow_parametric(df: pl.DataFrame) -> None:
     df_pa = df.to_arrow()
-    result = pl.from_dataframe(df_pa)
```

Comment:
these tests are very specific to the interchange protocol, so I've kept the test to those (rather than letting the pycapsule interface take precedence)